src/lang/af.json

```diff
@@ -33,6 +33,7 @@
   "app_tracked":"This user has visited recently, and replays of new matches will be automatically parsed.",
   "app_cheese_bought":"Cheese bought",
   "app_dotacoach":"Ask a Coach",
+  "app_pvgna":"Find a Guide",
```

Comment:
language strings only need to be added to the english file.  The rest will be translated later.